### DISCUSSION TITLE
GriefPrevention Config.yml

### DISCUSSION THREAD

**Discussion by FrostBlade5** [GriefPrevention/GriefPrevention#1788] [Q&A]
### Observed Behavior

This is not a bug report but a question.

Players on my server asked if they can get more claim blocks, so the only way for me to allow more than 2000 blocks is by increasing the default value? Also it says 100 blocks per hour, does that mean once they reached 2000 they can still get more each hour or is that just the limit?

And a really important question for me, is there any sort of "support" from griefprevention regarding groupmanager? I was planning on giving certain groups a larger amount of claim blocks, is that possible?

### Expected Behavior

I explained my issue above.

### Reproduction steps

I explained my issue above.

### Stack trace or error log

```shell
I explained my issue above.
```

### Server version

```shell
1.18.1
```

### GriefPrevention version

```shell
16.17.1
```

### Configuration

```yaml
I explained my issue above.
```

### Plugin list

```shell
grief prevention
groupmanager
essentialsx
worldedit
multiverse
```

### Running without GriefPrevention

- [ ] I attempted running the server without GriefPrevention installed.
- [ ] The problem does _not_ occur when GriefPrevention is removed from the server.

### Running with only GriefPrevention

- [ ] I attempted running only GriefPrevention on the server.
- [ ] The issue still occurs when GriefPrevention is the only plugin running.

### Running on a fresh, clean server installation

- [ ] I attempted testing for the issue on a new server.
- [ ] The issue still occurs on a new server.

### Using unmodified client

- [ ] I attempted testing for the issue with the vanilla client.
- [ ] The issue still occurs when using the vanilla client.

### We appreciate you taking the time to fill out a bug report!

- [X] I searched for similar issues before submitting this bug report.

**Comment by Jikoo** [comment]
Most of the answers to your questions are already [on the wiki](https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration#claim-limits).

> `Claims.InitialBlocks: 100`
> 
> The number of claim blocks a new player starts with. Note that if you allow automatic new player claims, those claims can still be larger than this number (the player will just go into negative claim blocks until he accrues more).
> 
> `Claims.BlocksAccruedPerHour: 100`
> 
> The number of claim blocks awarded to players for each hour of play time on your server. These are awarded gradually (about every 5 minutes), but only to players who aren't just standing around doing nothing (idling).
> 
> `Claims.MaxAccruedBlocks: 80000`
> 
> The maximum number of accrued claim blocks any player may amass. This number does not limit bonus claim blocks granted by administrators (see above).

Re: per-group limits:
[GPAccruals](https://github.com/TechFortress/GriefPrevention/wiki/Addons#gpaccruals) is an addon that allows for per-group accrual rates and limits.

**Reply by FrostBlade5** [reply]
> Most of the answers to your questions are already [on the wiki](https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration#claim-limits).
> 
> > `Claims.InitialBlocks: 100`
> > The number of claim blocks a new player starts with. Note that if you allow automatic new player claims, those claims can still be larger than this number (the player will just go into negative claim blocks until he accrues more).
> > `Claims.BlocksAccruedPerHour: 100`
> > The number of claim blocks awarded to players for each hour of play time on your server. These are awarded gradually (about every 5 minutes), but only to players who aren't just standing around doing nothing (idling).
> > `Claims.MaxAccruedBlocks: 80000`
> > The maximum number of accrued claim blocks any player may amass. This number does not limit bonus claim blocks granted by administrators (see above).
> 
> Re: per-group limits: [GPAccruals](https://github.com/TechFortress/GriefPrevention/wiki/Addons#gpaccruals) is an addon that allows for per-group accrual rates and limits.

Thanks for helping, i looked at the documenation but i dont understand how this can help me with group manager.
GPAccruals documenation only shows this:

# Show colors in console output (and log files)
color-logs: true

# Debug output to console (and log files)
debug-mode: false

# Language file (for translations)
language-file: lang-en.yml

# Amount of accrued claim blocks per hour and max cap, based on permissions.
# You can add/remove as many names here that you want.
# Permission nodes are in the format griefprevention.accruals.<type>.<name>
# Examples:
#     griefprevention.accruals.increment.faster
#     griefprevention.accruals.maxcap.more
accruals:
  increment:
    faster: 110
    fastest: 125
  maxcap:
    more: 5000
    most: 10000

So what do i do with this information? How does it help me with groupmanager? Do i have to create groups within griefpreventions?

**Reply by Jikoo** [reply]
> Thanks for helping, i looked at the documenation but i dont understand how this can help me with group manager. GPAccruals documenation only shows this:
> 
> ```yaml
> # Show colors in console output (and log files)
> color-logs: true
> 
> # Debug output to console (and log files)
> debug-mode: false
> 
> # Language file (for translations)
> language-file: lang-en.yml
> 
> # Amount of accrued claim blocks per hour and max cap, based on permissions.
> # You can add/remove as many names here that you want.
> # Permission nodes are in the format griefprevention.accruals..
> # Examples:
> # griefprevention.accruals.increment.faster
> # griefprevention.accruals.maxcap.more
> accruals:
>   increment:
>     faster: 110
>     fastest: 125
>   maxcap:
>     more: 5000
>     most: 10000
> ```
> 
> So what do i do with this information? How does it help me with groupmanager? Do i have to create groups within griefpreventions?

Configure as desired and add the permission nodes you create to your groups in GroupManager.
